lib/net/tns/client.rb

```diff
@@ -58,6 +58,8 @@ def self.get_status(opts={})
             # a data length that is greater than the available data.
           end
 
+          response = conn.send_and_receive(ResendPacket.new())
```

Comment:
I don't remember seeing this behavior before - where the client sends Resends to the server. Is this really new, really old, or just always there and I missed it? (Not that I've paid much attention to TNS status requests.)